### ISSUE TITLE
resource put failed

### ISSUE THREAD

**Issue by maxx-bachurin** [n7mobile/concourse-bitbucket-pr#5]
The example of the pipeline being used:

```
resource_types:
 - name: bitbucket-pr
   type: registry-image
   source:
     repository: n7mobile/concourse-bitbucket-pr

resources:
 - name: pull-request
   type: bitbucket-pr
   check_every: 1m
   source:
      workspace: ((bitbucket_project))
      slug: ((bitbucket_repo))
      username: ((bitbucket-username))
      password: ((bitbucket-password))
      debug: true

jobs:
  - name: check-pr
    serial: true
    plan:
    - get: pull-request
      version: every
      trigger: true
    - put: pull-request
      params:
         repo_path: pull-request
         action: set:commit.build.status
         status: INPROGRESS
         url: example.com

```

Resource put fails with the following error:

```
selected worker: concourse-worker-5995d5548d-gpv7f
calling out cmd wioth dest  /tmp/build/put
running command: %!w(*fmt.wrapError=&{resource/out: params invalid: resource/model: version path or name is empty 0xc00001c190})

```

**Comment by antrov** [comment]
Hi @maxx-bachurin! Thanks for sharing your issue. It looks like an error message from one of the previous versions of our resource. To be more specific, this setting changed on 10th Feb, commit [3a93714ecc653a40c8623e59d51d1e891d110ede](https://github.com/n7mobile/concourse-bitbucket-pr/commit/3a93714ecc653a40c8623e59d51d1e891d110ede). 

We suggest to upgrade docker image to the latest version or downgrade configuration of the pipeline to match syntax from February.

Docker image and repo lacks tags with specyfic version. We are aware of the mess it generates. Tag-versioning is on our roadmap.

**Comment by maxx-bachurin** [comment]
yes, it works. thanks

**Comment by aal89** [comment]
There's an inconsistency in the readme. It shows you this:

```yaml
resource_types:
  - name: bitbucket-pr
    type: registry-image
    source:
      repository: n7mobile/concourse-bitbucket-pr
```

but this pulls in the old version of concourse-bitbucket-pr. It should be this (note the repository value):

```yaml
resource_types:
  - name: bitbucket-pr
    type: registry-image
    source:
      repository: n7docker/concourse-bitbucket-pr
```

**Comment by antrov** [comment]
Fixed in 7a3ca0321768cde399fbf9dbb4b81436dec317bb
